scripts/quests/otherAreas/Knocking_on_Forbidden_Doors.lua

```diff
@@ -0,0 +1,279 @@
+-----------------------------------
+-- Knocking on Forbidden Doors
+-----------------------------------
+-- Log ID: 4, Quest ID: 78
+-- Enaremand       : !pos
+-- Fyi_Chalmwoh    : !pos -39.273 -16.000 70.126 249
+-- Mire Incense KI : 709
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/mannequins')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local ID = require('scripts/zones/Mhaura/IDs')
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.KNOCKING_ON_FORBIDDEN_DOORS)
+
+quest.sections =
+{
+    -- START: Talk to Enaremand (J-7) on the upper level in Tavnazian Safehold
+    -- QUEST AVAILABLE
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.BEHIND_THE_SMILE)
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Enaremand'] = quest:progressEvent(535),
+
+            onEventFinish =
+            {
+                [535] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    -- QUEST ACCEPTED
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Chemioue'] =
+            {
+                onTrigger = function(player, csid, option, npc)
+                    if quest:getVar(player, 'prog') == 0 then
+                        return quest:event(536)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [536] = function(player, csid, option, npc)
+                    quest:setVar(player, 'prog', 1)
+                end,
+            },
+        },
+
+        [xi.zone.PHOMIUNA_AQUEDUCTS] =
+        {
+            ['Wooden_Ladder'] =
```

Comment:
What you could do here is rename the specific Wooden_Ladder in npc_list.sql to Wooden_Ladder_Knocking or something, and then reference that specific ladder here.